boards/ek-lm4f120xl/include/periph_conf.h

```diff
@@ -99,6 +101,68 @@ extern "C" {
 #define ADC_NUMOF           (12)
 /** @} */
 
+/**
+ * @name SPI configuration
+ * @{
+ */
+static const struct spi_conf_s spi_confs[] = {
```

Comment:
@haukepetersen are not we forced to implement rather a `spi_conf_t` struct? Or are we free to describe spi devices as we need as long as we respect the API using `spi_t`? If so, I don't see why this should be called `spi_conf_s`.